rook/migration/v1.11.x-to-v1.14.x.md

```diff
@@ -0,0 +1,98 @@
+# Upgrade rook-ceph from v1.11.x to v1.14.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Apply upgrade to 1.12.x - _disruptive_
+
+- 1. Fetch chart version:
+
+```console
+rm -rf helmfile.d/upstream
+helm fetch rook-release/rook-ceph --version v1.12.11 --untar --destination helmfile.d/upstream
+helm fetch rook-release/rook-ceph-cluster --version v1.12.11 --untar --destination helmfile.d/upstream
+```
+
+- 2. Upgrade rook-ceph:
+
+```console
+helmfile -e workload -l app=operator diff
+helmfile -e workload -l app=operator apply
+helmfile -e workload -l app=cluster diff
+helmfile -e workload -l app=cluster apply
+```
```

Comment:
I would like to see this as two steps. And a comment between explaining why. 
Something like
```suggestion

To give time for the operator to update internal objects, such as CRDs.
We run the upgrade in two steps.

- 2. Upgrade rook-ceph operator:

```console
helmfile -e workload -l app=operator diff
helmfile -e workload -l app=operator apply

- 3. Upgrade rook-ceph cluster

Wait a short while for the operator, before upgrading the cluster.

```console
helmfile -e workload -l app=cluster diff
helmfile -e workload -l app=cluster apply
```
```

So it is clear to give the operator some time before upgrading ceph (the cluster)